image/src/mobilenet/custom-mobilenet.ts

```diff
@@ -207,7 +203,7 @@ export class CustomMobileNet {
  * @param modelOptions options determining what model to load
  */
 export async function loadTruncatedMobileNet(modelOptions?: ModelOptions) {
-    const [checkpointUrl, trainingLayer] = parseModelOptions(modelOptions);
+    const checkpointUrl = parseModelOptions(modelOptions);
```

Comment:
does it make sense for this to really have an interface at this point? It only can be a `checkpointURL: string` or a `alpha: number` and never both and is only used in this method. How about:

```ts
export async function loadTruncatedMobileNet(checkpointURL_or_alpha: string | number) {
    if (typeof checkpointURL_or_alpha === 'string') {...
```

Reply:
after discussing, I agree, maintaining `ModelOptions` will set us up better for future possibilities